video_xblock/video_xblock.py

```diff
@@ -749,6 +761,56 @@ def update_metadata_authentication(self, auth_data, player):
             self.metadata['client_id'] = ''  # Brightcove API
             self.metadata['client_secret'] = ''  # Brightcove API
 
+    @XBlock.handler
+    def upload_manual_transcript_handler(self, request, suffix=''):
+        """
+        Upload manual transcript for video xblock.
+
+        Saves transcript as the .vtt file.
+        Performs file conversion beforehand (if a different supported file type is received).
+        
+        Arguments:
+            request: Data from frontend on a transcript.
+            _suffix: Slug used for routing. Imposed by `XBlock.handler`.
+        Returns: Transcript data uploaded manually.
```

Comment:
```suggestion
```

I would remove it because it has no useful info

Reply:
done